src/style/site.css

```diff
@@ -576,7 +576,9 @@ ul.footer-icons li {
 .notes-place-holder {
   height: 548px;
   padding-top:175px;
-  background-color: #ecf0f1;;
+  background-color: #ecf0f1;
```

Comment:
Instead of adding this css, why don't we use the bootstrap class `well` on the div? That way we don't have to maintain it.